db/schema.rb

```diff
@@ -0,0 +1,34 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_230_815_104_158) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.text 'content'
+    t.bigint 'author_id'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_posts_on_author_id'
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.string 'photo'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
```

Comment:
- [ ] [REQUIRED] Good jobs o far! There should be migration files for creating `users`, `posts`, `comments`, `likes` but you create for only `posts` and `users`. Please create migration files for `comments` and `likes` contains about [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) 🙏🏼

Reply:
Hi @mukhammadrizooff, 

Thanks for highlighting the requirements. I will update them and submit another evaluation.